### ISSUE TITLE
If loading database and then without loading an image, click on blank space will cause abort

### ISSUE THREAD

**Issue by mingrui** [maubreville/SlideRunner#8]
Bug in action: https://drive.google.com/open?id=1BTUg0L8MyhnI-w85BPVp4cf6itA34jo9

```
$ python main.py
List of available plugins:
 Normalize (Macenko)   Version 0.1
      OTSU threshold   Version 0.1
Positive Pixel Count (Aperio)   Version 0.1
<Linux>
Traceback (most recent call last):
  File "/mnt/960EVO/workspace/pathology/SlideRunner/SlideRunner/gui/mouseEvents.py", line 374, in pressImage
    leftClickImage(self,event)
  File "/mnt/960EVO/workspace/pathology/SlideRunner/SlideRunner/gui/mouseEvents.py", line 144, in leftClickImage
    for annotation in self.annotationsFlags:
AttributeError: 'SlideRunnerUI' object has no attribute 'annotationsFlags'
[1]    28746 abort (core dumped)  python main.py

```

**Comment by maubreville** [comment]
Thanks for pointing out!

**Comment by maubreville** [comment]
Fixed with 86ca932

**Comment by mingrui** [comment]
nice! thank you!
